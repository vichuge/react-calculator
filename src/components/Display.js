import React from 'react';

const Display = () => (
  <div className="Display">
    <p>123</p>
  </div>
);

export default Display;
