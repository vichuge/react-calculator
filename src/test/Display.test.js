import React from 'react';
import renderer from 'react-test-renderer';
import Display from '../components/Display';

it('return display', () => {
  const tree = renderer
    .create(<Display result="0123" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
