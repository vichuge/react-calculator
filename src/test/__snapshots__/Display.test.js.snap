// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`return display 1`] = `
<div
  className="display"
>
  <p>
    0123
  </p>
</div>
`;
